resources/lang/de/injuries.php

```diff
@@ -23,10 +23,7 @@
     'Hamstring'                 => 'Sehnenzerrung',
     'Calf/Shin Injury'          => 'Schienbeinverletzung',
     'Hip/Thigh Injury'          => 'Hüftverletzung',
-    /*
-     * Note: I'm not sure what medical condition "suspended" describes. The literal translation is most likely wrong.
-     */
-    'Suspended'                 => 'Schwebend',
+    'Suspended'                 => 'Gesperrt',
```

Comment:
Are you sure that you call a player that is suspended due to a red car 'Gesperrt'? To me this sounds as a road that is blocked 🤣

Reply:
Yes, I'm sure ;)  And you are right to.
A road can be "gesperrt" but football players will be "gesperrt" too.

"Die Straße ist wegen Bauarbeiten gesperrt" => The road is closed due to construction.
"Der Spieler wurde wegen einer roten Karte für das nächste Spiel gesperrt" => The player is suspended for the next game due to a red card.